src/community/json-ld/src/main/java/org/geoserver/jsonld/expressions/ExpressionsUtils.java

```diff
@@ -251,4 +259,70 @@ public static String removeQuotes(String cqlFilter) {
         }
         return strBuilder.toString();
     }
+
+    /**
+     * Clean a CQL expression from the xpath function syntax to make the xpath suitable to be
+     * encoded as a PropertyName
+     *
+     * @param expression
+     * @param toReplace
+     * @param replacement
+     * @return
+     */
+    public static String cleanCQLExpression(
+            String expression, String toReplace, String replacement) {
+        if (expression.indexOf(XPATH_FUN_START) != -1)
```

Comment:
@taba90 I have a doubt here, if someone writes ``xpath  (`` (notice the extra spaces) in a CQL expression will this be valid?

Reply:
checked: an "xpath (" won't be considered valid

Reply:
ok, thx